cli/src/json-schemas/ContentDirectory.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  ChannelInputParameters,
+  VideoInputParameters,
+  VideoCategoryInputParameters,
+  ChannelCategoryInputParameters,
+  JsonSchema,
+} from '../Types'
+
+export const VideoCategoryInputSchema: JsonSchema<VideoCategoryInputParameters> = {
+  type: 'object',
+  additionalProperties: false,
+  properties: {
+    name: {
+      type: 'string',
+    },
+  },
+}
+
+export const ChannelCategoryInputSchema: JsonSchema<ChannelCategoryInputParameters> = VideoCategoryInputSchema
+
+export const ChannelInputSchema: JsonSchema<ChannelInputParameters> = {
+  type: 'object',
+  additionalProperties: false,
+  properties: {
+    category: { type: 'number' },
+    description: { type: 'string' },
+    isPublic: { type: 'boolean' },
+    language: { type: 'string' },
+    title: { type: 'string' },
+    coverPhotoPath: { type: 'string' },
+    avatarPhotoPath: { type: 'string' },
+    rewardAccount: { type: 'string' },
```

Comment:
the reward account is an `Option<AccountId>`  when creating a channel and `Option<Option<AccountId>>` when updating a channel. Is this properly captured by this type?

Reply:
I updated the type to `['string', 'null']` in https://github.com/Joystream/joystream/pull/2334/commits/2a8ddef4d1a8a5e6ccf288da0074da7eb9a14d40, so that `null` can represent empty value and no value (`undefined`) will imply skipping the field during update, but unfortunately there is a problem with encoding `Option<Option<null>>` using `@polkadot/api`